### PR TITLE
feat(UrlShortenerCommand): add url shortener command

### PR DESCRIPTION
**Pull request by eternalmoon1234**
This is a URL shortener command that calls TinyURL's API.

### CODE REVIEW

File: src/commands/Utility Commands/URLShortenerCommand.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+			},
+			message
+		)
+	);
+};
+
+export const name: string = 'urlshorten';
+export const category: string = 'utility';
+export const description: string = 'Shorten a long URL into a tinyurl';
+export const cooldown: number = 6000;
```
Comment (Milo123459) [inline src/commands/Utility Commands/URLShortenerCommand.ts:42]: set cooldown to 10000

Comment (eternalmoon1234) [reply src/commands/Utility Commands/URLShortenerCommand.ts:42]: ok

File: src/commands/Utility Commands/URLShortenerCommand.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		);
+	}
+
+	if (!args[0].startsWith('https://') || !args[0].startsWith('http://')) {
+		return message.channel.send(
+			client.embed({ description: 'You can only shorten URLs!' }, message)
+		);
+	}
+
+	message.delete();
```
Comment (Milo123459) [inline src/commands/Utility Commands/URLShortenerCommand.ts:24]: remove deleting the message

Comment (eternalmoon1234) [reply src/commands/Utility Commands/URLShortenerCommand.ts]: done

File: src/commands/Utility Commands/URLShortenerCommand.ts (hunk 1 above)
Comment (eternalmoon1234) [reply src/commands/Utility Commands/URLShortenerCommand.ts]: I didn't find any terms of service about the API that shows the rate limit you should use, so I think 10,000 is safe, so yeah

**Review by Milo123459 — APPROVED** [review]

**Pull request merged**